crates/db_schema/src/impls/review_comment.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{
+  newtypes::{PersonId, ReviewCommentId},
+  schema::review_comment::dsl::{approved, approver_id, review_comment, updated},
+  source::review_comment::{ReviewComment, ReviewCommentForm},
+  utils::{get_conn, naive_now, DbPool},
+};
+use diesel::{
+  dsl::{insert_into, update},
+  result::Error,
+  ExpressionMethods,
+  QueryDsl,
+};
+use diesel_async::RunQueryDsl;
+
+impl ReviewComment {
+  pub async fn create(pool: &DbPool, form: &ReviewCommentForm) -> Result<ReviewComment, Error> {
+    let conn = &mut get_conn(pool).await?;
+    insert_into(review_comment)
+      .values(form)
+      .get_result(conn)
+      .await
+  }
+
+  pub async fn approve(
+    pool: &DbPool,
+    review_id_: ReviewCommentId,
+    by_approver_id: PersonId,
```

Comment:
Should pass in a bool, so as to have a way to unapprove comments. Much in the same way that reports can be unresolved. 

Or, follow the convention of comment_report, and add an `unapprove` function.